lib/collection/src/operations/types.rs

```diff
@@ -858,6 +873,80 @@ impl From<VectorParams> for VectorsConfig {
     }
 }
 
+/// Update params of single vector data storage
+#[derive(Debug, Hash, Deserialize, Serialize, JsonSchema, Validate, Clone, PartialEq, Eq)]
+#[serde(rename_all = "snake_case")]
+pub struct UpdateVectorParams {
```

Comment:
Do we want to call this `UpdateVectorParams` (and `UpdateVectorsConfig` respectively), or `VectorParamsDiff` (and `VectorsConfigDiff`)?

Reply:
I vote for `VectorParamsDiff` because other such structs have similar name (for example, `HnswConfigDiff`)

Reply:
I have one objection against `VectorParamsDiff`.

All other '*diff' structs are mergeable, and have `Merge` and the `DiffConfig` trait. This one does not. That would make it inconsistent.

What do you think?

Reply:
I've restructured the internals a bit to make them compatible with `ConfigDiff`.

The above structures are now called `VectorParamsDiff` (and `VectorsConfigDiff`) respectively.